src/java/org/apache/cassandra/repair/RepairJob.java

```diff
@@ -197,6 +198,7 @@ private boolean isTransient(InetAddressAndPort ep)
                                                   InetAddressAndPort local,
                                                   Predicate<InetAddressAndPort> isTransient,
                                                   boolean isIncremental,
+                                                  boolean pushRepair,
```

Comment:
can a repair be simultaneously push and pull?

Reply:
added a check in `RepairOption` to prevent using both